Error: The results of prepareCache must be produced by glob against workPath

I'm trying to setup Now deployment for our Nuxt project (which builds/starts/runs dev fine) and hitting this error on Now:

```
----------------- Collect cache -----------------
--
12:58:15 AM | ℹ 0 files collected from .now_cache
12:58:20 AM | ℹ 30855 files collected from node_modules_dev
12:58:22 AM | ℹ 10100 files collected from node_modules_prod
12:58:22 AM | ℹ Collect cache took: 6785.539765 ms
12:58:22 AM | Build completed. Populating build cache...
12:58:22 AM | Error: The results of prepareCache must be produced by glob against workPath or prepareCachePath
12:58:22 AM | at packCache (/var/task/sandbox-worker.js:26382:15)
12:58:22 AM | at uploadCacheStep (/var/task/sandbox-worker.js:26347:28)
12:58:22 AM | at <anonymous>
12:58:22 AM | done
```

This seems loosely related to https://github.com/nuxt/now-builder/issues/111 but a different error message. This is what our now.json looks like:

```
{
  "version": 2,
  "builds": [{
    "src": "nuxt/indieshops/nuxt.config.js",
    "use": "@nuxtjs/now-builder",
    "config": {}
  }]
}
```

In case it helps, here is the full build log:

```

Installing build runtime...
--
01:10:07 AM | Build runtime installed: 3158.395ms
01:10:08 AM | Looking up build cache...
01:10:09 AM | ----------------- Prepare build -----------------
01:10:09 AM | Downloading files...
01:10:10 AM | Working directory: /tmp/1fa5afbc/nuxt/indieshops
01:10:10 AM | missing `engines` in `package.json`, using default range: 8.10.x
01:10:10 AM | Using npm
01:10:10 AM | ℹ Prepare build took: 993.855328 ms
01:10:10 AM | ----------------- Install devDependencies -----------------
01:10:10 AM | Running npm install
01:10:49 AM | > grpc@1.23.3 install /tmp/1fa5afbc/nuxt/indieshops/node_modules/grpc
01:10:49 AM | > node-pre-gyp install --fallback-to-build --library=static_library
01:10:49 AM | node-pre-gyp
01:10:49 AM |  
01:10:49 AM | WARN Using request for node-pre-gyp https download
01:10:50 AM | [grpc] Success: "/tmp/1fa5afbc/nuxt/indieshops/node_modules_dev/grpc/src/node/extension_binary/node-v57-linux-x64-glibc/grpc_node.node" is installed via remote
01:10:50 AM | > node-sass@4.12.0 install /tmp/1fa5afbc/nuxt/indieshops/node_modules/node-sass
01:10:50 AM | > node scripts/install.js
01:10:50 AM | Downloading binary from https://github.com/sass/node-sass/releases/download/v4.12.0/linux-x64-57_binding.node
01:10:51 AM | Download complete
01:10:51 AM | Binary saved to /tmp/1fa5afbc/nuxt/indieshops/node_modules_dev/node-sass/vendor/linux-x64-57/binding.node
01:10:51 AM | Caching binary to /tmp/.npm/node-sass/4.12.0/linux-x64-57_binding.node
01:10:51 AM | > core-js@3.2.1 postinstall /tmp/1fa5afbc/nuxt/indieshops/node_modules/@firebase/polyfill/node_modules/core-js
01:10:51 AM | > node scripts/postinstall \|\| echo "ignore"
01:10:52 AM | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
01:10:52 AM | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
01:10:52 AM | > https://opencollective.com/core-js
01:10:52 AM | > https://www.patreon.com/zloirock
01:10:52 AM | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
01:10:52 AM | > core-js@2.6.9 postinstall /tmp/1fa5afbc/nuxt/indieshops/node_modules/core-js
01:10:52 AM | > node scripts/postinstall \|\| echo "ignore"
01:10:52 AM | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
01:10:52 AM | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
01:10:52 AM | > https://opencollective.com/core-js
01:10:52 AM | > https://www.patreon.com/zloirock
01:10:52 AM | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
01:10:52 AM | > protobufjs@6.8.8 postinstall /tmp/1fa5afbc/nuxt/indieshops/node_modules/protobufjs
01:10:52 AM | > node scripts/postinstall
01:10:52 AM | > node-sass@4.12.0 postinstall /tmp/1fa5afbc/nuxt/indieshops/node_modules/node-sass
01:10:52 AM | > node scripts/build.js
01:10:52 AM | Binary found at /tmp/1fa5afbc/nuxt/indieshops/node_modules_dev/node-sass/vendor/linux-x64-57/binding.node
01:10:52 AM | Testing binary
01:10:52 AM | Binary is fine
01:10:52 AM | > nuxt@2.9.2 postinstall /tmp/1fa5afbc/nuxt/indieshops/node_modules/nuxt
01:10:52 AM | > opencollective \|\| exit 0
01:10:53 AM |  
01:10:53 AM | :-:
01:10:53 AM | .==-+:
01:10:53 AM | .==. :+- .-=-
01:10:53 AM | .==.   :==++-+=.
01:10:53 AM | :==.     -**: :+=.
01:10:53 AM | :+-      :*+++. .++.
01:10:53 AM | :+-      -*= .++: .=+.
01:10:53 AM | -+:      =*-   .+*: .=+:
01:10:53 AM | -+:     .=*-     .=*-  =+:
01:10:53 AM | .==:     .+*:        -*-  -+-
01:10:53 AM | .=+:.....:+*-.........:=*=..=*-
01:10:53 AM | .-=------=++============++====:
01:10:53 AM |  
01:10:53 AM | Thanks for installing nuxtjs
01:10:53 AM | Please consider donating to our open collective
01:10:53 AM | to help us maintain this package.
01:10:53 AM |  
01:10:53 AM | Number of contributors: 214
01:10:53 AM | Number of backers: 251
01:10:53 AM | Annual budget: US$ 66,796
01:10:53 AM | Current balance: US$ 3,197
01:10:53 AM |  
01:10:53 AM | Donate: https://opencollective.com/nuxtjs/donate
01:10:53 AM |  
01:10:55 AM | npm
01:10:55 AM | WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.0.7 (node_modules/@nuxt/builder/node_modules/fsevents):
01:10:55 AM | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.0.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
01:10:55 AM | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/fsevents):
01:10:55 AM | npm
01:10:55 AM | WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
01:10:55 AM | added 1546 packages in 45.345s
01:10:55 AM | ℹ Install devDependencies took: 45877.980358 ms
01:10:55 AM | ----------------- Nuxt build -----------------
01:10:55 AM | Running nuxt build --standalone --no-lock --config-file "nuxt.config.js"
01:10:58 AM | ℹ Production build
01:10:58 AM | ✔ Builder initialized
01:10:58 AM | ✔ Nuxt files generated
01:11:03 AM | ℹ Compiling Client
01:11:03 AM | ℹ Starting type checking service...
01:11:03 AM | ℹ Using 1 worker with 4096MB memory limit
01:11:03 AM | ℹ Starting type checking service...
01:11:03 AM | ℹ Using 1 worker with 4096MB memory limit
01:11:37 AM | ✔ Client: Compiled successfully in 33.28s
01:11:37 AM | ℹ Compiling Server
01:12:15 AM | ✔ Server: Compiled successfully in 38.55s
01:12:15 AM | Hash: d5e0d07a9098b7e984d1
01:12:15 AM | Version: webpack 4.40.2
01:12:15 AM | Time: 33286ms
01:12:15 AM | Built at: 2019-09-26 08:11:37
01:12:15 AM | Asset       Size  Chunks                                Chunk Names
01:12:15 AM | ../server/client.manifest.json   16.7 KiB          [emitted]
01:12:15 AM | 03d8579493255f755181.js  719 bytes       5  [emitted] [immutable]         pages/feedback
01:12:15 AM | 1cfd920c3fb3d9df0f4d.js   5.21 KiB       6  [emitted] [immutable]         pages/index
01:12:15 AM | 2e39e82dbda471c86560.js   4.91 KiB       3  [emitted] [immutable]         pages/category/_id
01:12:15 AM | 525367db1fc3af83b6d5.js    304 KiB       0  [emitted] [immutable]  [big]  pages/category/_id.pages/index
01:12:15 AM | 5689f3b89984ab34ecca.js    603 KiB       1  [emitted] [immutable]  [big]  app
01:12:15 AM | 836ce1a1c15bbe651242.js  719 bytes       7  [emitted] [immutable]         pages/suggest
01:12:15 AM | 9a943b6e5242a268dfc8.js    4.7 KiB       2  [emitted] [immutable]         pages/about
01:12:15 AM | LICENSES      3 KiB          [emitted]
01:12:15 AM | e21d046094fa674e1822.js   2.45 KiB       8  [emitted] [immutable]         runtime
01:12:15 AM | eccb67d4170469784d62.js  199 bytes       4  [emitted] [immutable]         pages/category/index
01:12:15 AM | img/59eb2da.png   14.6 KiB          [emitted]
01:12:15 AM | + 2 hidden assets
01:12:15 AM | Entrypoint app = e21d046094fa674e1822.js 5689f3b89984ab34ecca.js
01:12:15 AM | WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
01:12:15 AM | This can impact web performance.
01:12:15 AM | Assets:
01:12:15 AM | 525367db1fc3af83b6d5.js (304 KiB)
01:12:15 AM | 5689f3b89984ab34ecca.js (603 KiB)
01:12:15 AM | Hash: 850317a71779c46eeb4b
01:12:15 AM | Version: webpack 4.40.2
01:12:15 AM | Time: 38550ms
01:12:15 AM | Built at: 2019-09-26 08:12:15
01:12:15 AM | Asset       Size  Chunks                         Chunk Names
01:12:15 AM | 3402c6507fd3b07fb5a9.js   4.85 KiB       1  [emitted] [immutable]  pages/about
01:12:15 AM | 4dac2f42b9cc06755930.js  228 bytes       3  [emitted] [immutable]  pages/category/index
01:12:15 AM | 5406f59966542cacf57d.js  657 bytes       6  [emitted] [immutable]  pages/suggest
01:12:15 AM | c62daf3d072c105cd499.js    531 KiB       2  [emitted] [immutable]  pages/category/_id
01:12:15 AM | e33eb31380dd833d6ac4.js    529 KiB       5  [emitted] [immutable]  pages/index
01:12:15 AM | e8006b7ba90ed6489e7d.js  657 bytes       4  [emitted] [immutable]  pages/feedback
01:12:15 AM | server.js    932 KiB       0  [emitted]              app
01:12:15 AM | server.manifest.json  843 bytes          [emitted]
01:12:15 AM | + 7 hidden assets
01:12:15 AM | Entrypoint app = server.js server.js.map
01:12:15 AM | WARNING in ./node_modules_dev/grpc/src/grpc_extension.js 32:12-33
01:12:15 AM | Critical dependency: the request of a dependency is an expression
01:12:15 AM | @ ./node_modules_dev/grpc/index.js
01:12:15 AM | @ ./node_modules_dev/@firebase/firestore/dist/index.node.cjs.js
01:12:15 AM | @ ./node_modules_dev/firebase/firestore/dist/index.cjs.js
01:12:15 AM | @ ./plugins/firebase.ts
01:12:15 AM | @ ./.nuxt/index.js
01:12:15 AM | @ ./.nuxt/server.js
01:12:15 AM | @ multi ./.nuxt/server.js
01:12:15 AM | WARNING in ./node_modules_dev/grpc/node_modules/node-pre-gyp/lib/pre-binding.js 20:22-48
01:12:15 AM | Critical dependency: the request of a dependency is an expression
01:12:15 AM | @ ./node_modules_dev/grpc/src/grpc_extension.js
01:12:15 AM | @ ./node_modules_dev/grpc/index.js
01:12:15 AM | @ ./node_modules_dev/@firebase/firestore/dist/index.node.cjs.js
01:12:15 AM | @ ./node_modules_dev/firebase/firestore/dist/index.cjs.js
01:12:15 AM | @ ./plugins/firebase.ts
01:12:15 AM | @ ./.nuxt/index.js
01:12:15 AM | @ ./.nuxt/server.js
01:12:15 AM | @ multi ./.nuxt/server.js
01:12:15 AM | WARNING in ./node_modules_dev/grpc/node_modules/node-pre-gyp/lib/util/versioning.js 17:20-67
01:12:15 AM | Critical dependency: the request of a dependency is an expression
01:12:15 AM | @ ./node_modules_dev/grpc/node_modules/node-pre-gyp/lib/pre-binding.js
01:12:15 AM | @ ./node_modules_dev/grpc/src/grpc_extension.js
01:12:15 AM | @ ./node_modules_dev/grpc/index.js
01:12:15 AM | @ ./node_modules_dev/@firebase/firestore/dist/index.node.cjs.js
01:12:15 AM | @ ./node_modules_dev/firebase/firestore/dist/index.cjs.js
01:12:15 AM | @ ./plugins/firebase.ts
01:12:15 AM | @ ./.nuxt/index.js
01:12:15 AM | @ ./.nuxt/server.js
01:12:15 AM | @ multi ./.nuxt/server.js
01:12:15 AM | WARNING in ./node_modules_dev/bytebuffer/dist/bytebuffer-node.js
01:12:15 AM | Module not found: Error: Can't resolve 'memcpy' in '/tmp/1fa5afbc/nuxt/indieshops/node_modules_dev/bytebuffer/dist'
01:12:15 AM | @ ./node_modules_dev/bytebuffer/dist/bytebuffer-node.js
01:12:15 AM | @ ./node_modules_dev/grpc/node_modules/protobufjs/dist/protobuf.js
01:12:15 AM | @ ./node_modules_dev/grpc/index.js
01:12:15 AM | @ ./node_modules_dev/@firebase/firestore/dist/index.node.cjs.js
01:12:15 AM | @ ./node_modules_dev/firebase/firestore/dist/index.cjs.js
01:12:15 AM | @ ./plugins/firebase.ts
01:12:15 AM | @ ./.nuxt/index.js
01:12:15 AM | @ ./.nuxt/server.js
01:12:15 AM | @ multi ./.nuxt/server.js
01:12:15 AM | ℹ Nuxt build took: 80003.975186 ms
01:12:15 AM | ----------------- Install dependencies -----------------
01:12:15 AM | Running npm install
01:12:25 AM | > grpc@1.23.3 install /tmp/1fa5afbc/nuxt/indieshops/node_modules/grpc
01:12:25 AM | > node-pre-gyp install --fallback-to-build --library=static_library
01:12:25 AM | node-pre-gyp
01:12:25 AM |  
01:12:25 AM | WARN Using needle for node-pre-gyp https download
01:12:26 AM | [grpc] Success: "/tmp/1fa5afbc/nuxt/indieshops/node_modules_prod/grpc/src/node/extension_binary/node-v57-linux-x64-glibc/grpc_node.node" is installed via remote
01:12:26 AM | > core-js@3.2.1 postinstall /tmp/1fa5afbc/nuxt/indieshops/node_modules/@firebase/polyfill/node_modules/core-js
01:12:26 AM | > node scripts/postinstall \|\| echo "ignore"
01:12:26 AM | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
01:12:26 AM | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
01:12:26 AM | > https://opencollective.com/core-js
01:12:26 AM | > https://www.patreon.com/zloirock
01:12:26 AM | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
01:12:26 AM | > protobufjs@6.8.8 postinstall /tmp/1fa5afbc/nuxt/indieshops/node_modules/protobufjs
01:12:26 AM | > node scripts/postinstall
01:12:27 AM | added 431 packages in 10.799s
01:12:27 AM | ℹ Install dependencies took: 11294.862802 ms
01:12:27 AM | ----------------- Collect artifacts -----------------
01:12:44 AM | ℹ Collect artifacts took: 17033.796282 ms
01:12:48 AM | Cache dir: /tmp/5f525c5b/nuxt/indieshops
01:12:48 AM | ----------------- Clean cache -----------------
01:12:48 AM | ℹ Clean cache took: 6.794641 ms
01:12:48 AM | ----------------- Collect cache -----------------
01:12:48 AM | ℹ 0 files collected from .now_cache
01:12:53 AM | ℹ 30855 files collected from node_modules_dev
01:12:55 AM | ℹ 10100 files collected from node_modules_prod
01:12:55 AM | ℹ Collect cache took: 6740.987073 ms
01:12:55 AM | Build completed. Populating build cache...
01:12:55 AM | Error: The results of prepareCache must be produced by glob against workPath or prepareCachePath
01:12:55 AM | at packCache (/var/task/sandbox-worker.js:26382:15)
01:12:55 AM | at uploadCacheStep (/var/task/sandbox-worker.js:26347:28)
01:12:55 AM | at <anonymous>
01:12:55 AM | done
```

--- a/src/prepare-cache.ts
+++ b/src/prepare-cache.ts
@@ -6,10 +6,10 @@ import consola from 'consola'
 import { startStep, endStep } from './utils'
 
 // Amend below line once https://github.com/zeit/now/issues/2992 is resolved
-async function prepareCache ({ prepareCachePath, workPath, entrypoint }: PrepareCacheOptions & { prepareCachePath: string }): Promise<Record<string, FileRef>> {
+async function prepareCache ({ cachePath, workPath, entrypoint }: PrepareCacheOptions): Promise<Record<string, FileRef>> {
   const entryDir = path.dirname(entrypoint)
   const rootDir = path.join(workPath, entryDir)
-  const cacheDir = path.join(prepareCachePath, entryDir)
+  const cacheDir = path.join(cachePath, entryDir)
 
   consola.log('Cache dir:', cacheDir)
 

--- a/test/utils/run-build-lambda.js
+++ b/test/utils/run-build-lambda.js
@@ -43,7 +43,7 @@ async function runBuildLambda (inputPath) {
   })
 
   const cacheResult = await wrapper.prepareCache({
-    prepareCachePath: path.join(workPath, '.cache'),
+    cachePath: path.join(workPath, '.cache'),
     workPath,
     entrypoint
   })

